### PR TITLE
DOC: some updates to tutorial

### PR DESCRIPTION
**Pull request by dengemann**
@agramfort @Eric89GXL

### CODE REVIEW

File: doc/source/python_tutorial.rst [hunk 1]
```diff
@@ -26,17 +25,14 @@ What you can do with MNE Python
...
     - **Non-parametric statistics** in time, space and frequency (including cluster-level)
     - **Scripting** (batch and parallel computing)
 
-What you're not supposed to do with MNE Python
-----------------------------------------------
-
-	- **Boundary Element Modeling** use MNE and Freesurfer.
-
-
-.. note:: Package based on the FIF file format from Neuromag but can work with CTF and 4D after conversion to FIF.
```
Comment (Eric89GXL) [inline doc/source/python_tutorial.rst]: I think it's okay to leave this part in. It's true that people should look to FreeSurfer and MNE-C to set these things up (once per subject)

File: doc/source/python_tutorial.rst [hunk 2]
```diff
@@ -4,10 +4,9 @@
 Tutorial: MEG and EEG data processing with MNE and Python
 =========================================================
 
-Python offers transparent scripting on top of MNE.
-It was designed to be an alternative to the MNE matlab toolbox
-but now it can do much more (customize events, compute
-contrasts, statistics, time-frequency analysis etc.)
+MNE-Python reimplements most of MNE's functionality and offers transparent scripting.
```
Comment (Eric89GXL) [inline doc/source/python_tutorial.rst]: most of MNE-C's functionality, I think that's the language we settled on anyway

**Comment by agramfort** [comment]
LGTM

merge when you're done

File: doc/source/python_tutorial.rst [hunk 3]
```diff
@@ -33,10 +35,10 @@ What you can do with MNE Python
 What you're not supposed to do with MNE Python
 ----------------------------------------------
 
-	- **Boundary Element Modeling** use MNE and Freesurfer.
+    - **Boundary Element Modeling** use MNE and Freesurfer.
 
 
-.. note:: Package based on the FIF file format from Neuromag but can work with CTF and 4D after conversion to FIF.
+.. note:: Package based on the FIF file format from Neuromag. It can read and convert CTF, BTI/4D,, KIT and various EEG formats to FIF.
```
Comment (teonlamont) [inline doc/source/python_tutorial.rst]: BTI/4D,, KIT --> BTI/4D, KIT

File: doc/source/python_tutorial.rst [hunk 4]
```diff
@@ -386,7 +388,8 @@ What else can you do?
     - morph stc from one brain to another for group studies
     - compute mass univariate statistics base on custom contrasts
     - visualize source estimates
-    - export raw, epochs, and evoked data to other python data analysis libraries i.e. pandas and nitime
+    - export raw, epochs, and evoked data to other python data analysis libraries i.e. pandas
```
Comment (teonlamont) [inline doc/source/python_tutorial.rst]: i.e. --> e.g.

File: doc/source/python_tutorial.rst [hunk 5]
```diff
@@ -386,7 +388,8 @@ What else can you do?
...
+    - and many more thhings ...
```
Comment (teonlamont) [inline doc/source/python_tutorial.rst]: thhings --> things

**Comment by Eric89GXL** [comment]
Done? If so I'll run tests locally and merge.

**Comment by Eric89GXL** [comment]
(I killed your CI builds)

**Comment by dengemann** [comment]
done eric!

**Pull request merged**